lib/profiler/get_tick.h

```diff
@@ -67,6 +67,15 @@ get_tick(void)
     return regs;
 }
 
+#elif defined(HAVE_CLOCK_GETTIME_NSEC_NP) && defined(CLOCK_MONOTONIC_RAW)
+#include <ctime>
```

Comment:
This `#include` should be at the beginning of this header file. Otherwise, `CLOCK_MONOTONIC_RAW` will probably be undefined (in some environments) in some Profiler.h contexts. Please move it right after the existing `#if USE_XPROF_STATS` check.